### ISSUE TITLE
(Feature request) Allow Skytube to play other downloaded videos

### ISSUE THREAD

**Issue by Shapitizer** [ram-on/SkyTube#622]
I know Skytube is a Youtube client, but it would be useful if I could put other videos in the Skytube downloads folder so I could play them to my Chromecast

**Comment by atomjack** [comment]
Well currently if you're casting a video it plays the video off of the internet, even if you have the video downloaded onto your android device. In order to have the Chromecast play from your device, we'd have to be running a web server that could serve it up.

**Comment by Shapitizer** [comment]
Could that be implemented? I assume that's how VLC does it

**Comment by gzsombor** [comment]
Everything can be implemented if you have the time and resources :wink:

**Comment by Shapitizer** [comment]
True. I'm asking since currently I use VLC for casting local videos and Skytube for casting youtube videos, so I was hoping I'd just be able to put local videos into the same download folder that Skytube uses so I only need one app. No problem if it isn't going to be added as a feature
